src/yachtzee.ts

```diff
@@ -1003,8 +1003,15 @@ function yachtzeePotionSetup(yachtzeeTurns: number, simOnly?: boolean): number {
   return totalProfits;
 }
 
-function leprechaunMeatBonus(wt: number): number {
-  return 2 * wt + Math.sqrt(220 * wt) - 6;
+function bestFamUnderwaterGear(fam: Familiar): Item {
```

Comment:
This is fine now that it's sliced out, but in the future, we try not to have ternary chains that are _that_ long. A nicer structure for this would be to use nested if statements and several returns.

Reply:
Will for sure not block a merge, though